aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -192,6 +193,14 @@ def define(cls, spec: CalcJobProcessSpec):
             help='Files that are retrieved by the daemon will be stored in this node. By default the stdout and stderr '
                  'of the scheduler will be added, but one can add more by specifying them in `CalcInfo.retrieve_list`.')
 
+        # Errors caused or returned by the scheduler
```

Comment:
@sphuber sorry only saw this now: should some (all?) off these exit codes set `invalidates_cache=True`?

Reply:
Good point, I think they probably should. Will make a PR to correct it, we are now testing the implementation of the scheduler output parser for the SLURM plugin

Reply:
Yeah.. I'm pretty sure about the `100` error. For `110` and `120`, I think we should check if the inputs related to how much memory / walltime is requested go into the caching hash.
If they do not, `invalidates_cache=True` seems right to me. If they do, `False` is probably right - because the job is unlikely to succeed with the _same_ resources.